database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,12 +15,14 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('is_admin');
+            $table->string('is_active');
+            $table->string('username');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
             $table->rememberToken();
-            $table->timestamps();
+            $table->timestamps();          
```

Comment:
sao tab vao nhieu the nay???? :v

Reply:
fix xong comment nay merged lun nha bn

Reply:
cái này code mặc định nó v a ạ

Reply:
vâng ạ

Reply:
![image](https://user-images.githubusercontent.com/109664408/211734455-2fd38915-c5dc-49ba-9e88-92cf48f47e75.png)
a ơi e thấy trong code tab vừa đủ ạ